zqd/space/space.go

```diff
@@ -116,10 +147,28 @@ type config struct {
 	PacketPath string `json:"packet_path"`
 }
 
+type info struct {
+	MinTime nano.Ts `json:"min_time"`
+	MaxTime nano.Ts `json:"max_time"`
+}
+
+func (s Space) SetTimes(minTs, maxTs nano.Ts) error {
+	cur, err := loadInfo(s.path)
+	if err != nil {
+		if !os.IsNotExist(err) {
+			return err
+		}
+		cur = info{nano.MaxTs, nano.MinTs}
+	}
+	cur.MinTime = nano.Min(cur.MinTime, minTs)
+	cur.MaxTime = nano.Max(cur.MaxTime, maxTs)
+	return cur.save(s.path)
+}
+
 // loadConfig loads the contents of config.json in a space's path.
-func loadConfig(name string) (config, error) {
+func loadConfig(spacePath string) (config, error) {
 	var c config
-	b, err := ioutil.ReadFile(filepath.Join(name, configFile))
+	b, err := ioutil.ReadFile(filepath.Join(spacePath, configFile))
```

Comment:
Fine, but outside the scope of this pr.

Reply:
Fair enough, but since I added a similar function with the more appropriate argument name ("spacePath" vs "name"), I thought it was reasonable to adjust the pre-existing function for symmetry.

That said, is this a request to remove it?

Reply:
Nah, just pointing it out.

Reply:
👍